clients/fides-js/src/lib/gpp/string-to-consent.ts

```diff
@@ -209,7 +209,8 @@ export const fidesStringToConsent = ({
   fidesString,
   cmpApi,
 }: FidesStringToConsentArgs) => {
-  if (!fidesString || !cmpApi) {
+  const { gpp: gppString }: DecodedFidesString = decodeFidesString(fidesString);
+  if (!fidesString || !gppString || !cmpApi) {
```

Comment:
Minor optimization here, no reason to continue decoding the GPP string if one wasn't passed in.